### PR TITLE
Add calculated properties SN-455

### PR DESCRIPTION
**Pull request by vasiliy-chefonov**
https://saritasa.atlassian.net/browse/SN-455

### CODE REVIEW

File: src/Saritasa.NetForge.DomainServices/Extensions/EntityMetadataExtensions.cs
Comment (toanphamsaritasa) [inline src/Saritasa.NetForge.DomainServices/Extensions/EntityMetadataExtensions.cs:1]: Please fix the warnings on build.

**Review by toanphamsaritasa — APPROVED** [review]

File: demo/Saritasa.NetForge.Demo.Net7/Models/Product.cs [hunk 1]
```diff
@@ -68,5 +68,10 @@ public class Product
     /// The list of tags associated with the product.
     /// </summary>
     public List<ProductTag> Tags { get; set; } = new();
+
+    /// <summary>
+    /// Gets the volume of the project calculated by multiplying its length, width, and height in centimeters.
```
Comment (AlexanderMakarov124) [inline demo/Saritasa.NetForge.Demo.Net7/Models/Product.cs:73]: Gets the volume of the `product`?

File: src/Saritasa.NetForge.Domain/Entities/Options/EntityOptions.cs [hunk 2]
```diff
@@ -42,4 +42,9 @@ public EntityOptions(Type entityType)
     /// Options for properties of entity.
     /// </summary>
     public ICollection<PropertyOptions> PropertyOptions { get; set; } = new List<PropertyOptions>();
+
+    /// <summary>
+    /// Collection of the calculated property names.
+    /// </summary>
+    public List<string> CalculatedPropertiesNames { get; } = new();
```
Comment (AlexanderMakarov124) [inline src/Saritasa.NetForge.Domain/Entities/Options/EntityOptions.cs:49]: Double pluralizing, the name can be `CalculatedPropertyNames`

File: demo/Saritasa.NetForge.Demo.Net7/Infrastructure/Admin/UserAdminConfiguration.cs [hunk 3]
```diff
@@ -0,0 +1,15 @@
+﻿using Saritasa.NetForge.Demo.Net7.Models;
+using Saritasa.NetForge.DomainServices;
+using Saritasa.NetForge.DomainServices.Interfaces;
+
+namespace Saritasa.NetForge.Demo.Net7.Infrastructure.Admin;
+
+public class UserAdminConfiguration : IEntityAdminConfiguration<User>
```
Comment (AlexanderMakarov124) [inline demo/Saritasa.NetForge.Demo.Net7/Infrastructure/Admin/UserAdminConfiguration.cs:7]: Missing comment

**Review by AlexanderMakarov124 — APPROVED** [review]

**Pull request merged**